[2022-08-18 02:38:35,622] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-17T00:00:00+00:00 [queued]>
[2022-08-18 02:38:35,637] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.link_dag scheduled__2022-08-17T00:00:00+00:00 [queued]>
[2022-08-18 02:38:35,638] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 02:38:35,639] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-18 02:38:35,639] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-18 02:38:35,662] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): link_dag> on 2022-08-17 00:00:00+00:00
[2022-08-18 02:38:35,671] {standard_task_runner.py:52} INFO - Started process 8074 to run task
[2022-08-18 02:38:35,679] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'link_dag', 'scheduled__2022-08-17T00:00:00+00:00', '--job-id', '207', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmp6_33dz3u', '--error-file', '/tmp/tmpu1k5dkfy']
[2022-08-18 02:38:35,684] {standard_task_runner.py:80} INFO - Job 207: Subtask link_dag
[2022-08-18 02:38:35,880] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.link_dag scheduled__2022-08-17T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-18 02:38:36,064] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=link_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-17T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-17T00:00:00+00:00
[2022-08-18 02:39:18,094] {python.py:173} INFO - Done. Returned value was: None
[2022-08-18 02:39:18,117] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=link_dag, execution_date=20220817T000000, start_date=20220818T023835, end_date=20220818T023918
[2022-08-18 02:39:18,171] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-18 02:39:18,232] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
